python-pytest.el

```diff
@@ -119,41 +119,50 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
 (fmakunbound 'python-pytest-popup)
 (makunbound 'python-pytest-popup)
 
+(transient-define-argument python-pytest-popup:--tb ()
```

Comment:
is this an established or required naming convention? it's quite unlike all the other helpers.

Reply:
I've noticed it over [a few places in `magit`](https://github.com/magit/magit/blob/798aff56457aef4d1896db8129ab56d08ae12066/lisp/magit-log.el#L555-L575) and used that convention. I can update if you think it should be different or inline helper.

Reply:
okay, so it's a somewhat established convention. 👍 let's stick to it.

and for helper functions, i really prefer named helpers over anonymous inline lambdas and the like, so this is fine